test/pthread_extras/fib3_false.c

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
These are basically a bunch of tests that I wrote that I decided to exclude from the main folder that exercises the pthread library.

Most of these simply take a LONG time to run, and so didn't really seem appropriate for our general `test/pthread/` folder, but I didn't want to get rid of them straight away because they contain interesting performance bugs that should be worked out.

This certainly doesn't need to be here for any particular reason, but I didn't really know what to do with these examples.

Do you have any ideas on what we should do with these?